cloud/services/virtualnetworks/virtualnetworks_test.go

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
@devigned PTAL

Reply:
The tests are super clear about the expected behavior. Awesome!